packages/paste-core/core-bundle/package.json

```diff
@@ -19,10 +19,11 @@
   "scripts": {
     "prebuild": "node ./tools/generate.js",
     "build": "yarn clean && yarn compile",
-    "build:dev": "yarn clean && yarn compile:dev",
+    "build:dev": "yarn build",
+    "build:watch": "yarn clean && yarn compile:dev",
```

Comment:
Out of curiosity, why id core getting a watch script, and not any other package?

Reply:
Good catch, I'm not sure how that happened.. that change was meant to be in the design-tokens package! Fixed.